ubr/conf.py

```diff
@@ -38,12 +38,17 @@
 _handler.setLevel(logging.DEBUG)
 _handler.setFormatter(logging.Formatter('%(levelname)s - %(asctime)s - %(message)s'))
 
+_filehandler = logging.FileHandler('ubr.log')
```

Comment:
good that we gain a log file. Is this file created by builder-base-formula? It should be possible to do it there giving it the right permissions and ownership, eventually it would need to be rotated and so on